docs/dbaas/olm.md

```diff
@@ -0,0 +1,113 @@
+---
+slug: 'dbaas'
+---
+
+## How OLM (Operator Lifecycle Manager) works.
+
+DBaaS leverages the installation and upgrade of operators on OLM.
+To install an operator, we need to create an operator group and a subscription.
+The official documentation with detailed examples can be found [here](https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/).
+
+DBaaS installs several operators by default:
+- OLM 
+- DBaaS
+- PSMDB
+- PXC
+
+You can manually list the subscriptions using kubectl:
+```
+kubectl get subscriptions
+NAME                                 PACKAGE                           SOURCE                  CHANNEL
+my-percona-server-mongodb-operator   percona-server-mongodb-operator   operatorhubio-catalog   stable
+my-percona-xtradb-cluster-operator   percona-xtradb-cluster-operator   operatorhubio-catalog   stable
+```
+
+### Known issue.
+If you have 2 or more operators pending for installation approval, the second subscription OLM creates
+will have both operators. Listing the install plans could be confusing. For example:
+```
+kubectl get installplans
+NAME            CSV                                       APPROVAL   APPROVED
+install-9rxvz   percona-server-mongodb-operator.v1.13.1   Manual     false
+install-mghbh   percona-server-mongodb-operator.v1.13.1   Manual     false
+```
+It seems like both install plans are for PSMDB but examining the install plans individually can shed some light on the matter:
```

Comment:
I don't think this sentence is required.

Reply:
In my opinion, it is necessary. It describes the actual issue.

Reply:
@rasika-chivate can we keep this or can you help me to rephrase it?
In my opinion it is necessary because it describes the root cause of the problem.

Reply:
Can we rephrase to something" Although PSMDB appears in both install plans, examining each plan individually can provide insight into the scenario:"